unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/SettingsControllers/SpecificControllers/ShaderQualityControlController.cs

```diff
@@ -0,0 +1,48 @@
+using DCL.SettingsCommon.SettingsControllers.BaseControllers;
+using System;
+using UnityEngine;
+using ShaderQuality = DCL.SettingsCommon.QualitySettings.ShaderQuality;
+
+namespace MainScripts.DCL.Controllers.Settings.SettingsControllers.SpecificControllers
+{
+    [CreateAssetMenu(menuName = "Settings/Controllers/Controls/Shader Quality", fileName = "ShaderQualityControlController")]
+    public class ShaderQualityControlController : SpinBoxSettingsControlController
+    {
+        [SerializeField] private FloatVariable blurSize;
+        [SerializeField] private FloatVariable blurSigma;
+        [SerializeField] private FloatVariable outlineThickness;
+
+        private static readonly int GAUSSIAN_BLUR_KERNEL_SIZE = Shader.PropertyToID("gaussianBlurQuality");
+
+        public override object GetStoredValue() =>
+            (int)currentQualitySetting.shaderQuality;
+
+        public override void UpdateSetting(object newValue)
+        {
+            currentQualitySetting.shaderQuality = (ShaderQuality)newValue;
+
+            switch (currentQualitySetting.shaderQuality)
+            {
+                case ShaderQuality.LOW:
+                    Shader.SetGlobalInt(GAUSSIAN_BLUR_KERNEL_SIZE, 4);
```

Comment:
Can we incorporate these values in some quality presets?

Reply:
ShaderQuality is already on the quality preset, or you mean something else?